### PR TITLE
Main README update

### PR DESCRIPTION
**Pull request by taleldayekh**

### CODE REVIEW

**Comment by taleldayekh** [comment]
@christianbrb We have the section `Try Out the Raiden Demo dApp` with instructions on how to use the dApp on testnets. It is short but to the point and works well in the readme. I think we should keep it as it is and complement with a similar one but for mainnet? What do you think?

**Comment by taleldayekh** [comment]
Also, since we are mentioning the dApp and SDK in the main README it makes sense to add a section about the CLI/API also in the main README?

**Comment by christianbrb** [comment]
> 
> 
> @christianbrb We have the section `Try Out the Raiden Demo dApp` with instructions on how to use the dApp on testnets. It is short but to the point and works well in the readme. I think we should keep it as it is and complement with a similar one but for mainnet? What do you think?

I would keep this section. Topics for updating the readmes for mainnet can be put here #1785

**Comment by christianbrb** [comment]
> 
> 
> Also, since we are mentioning the dApp and SDK in the main README it makes sense to add a section about the CLI/API also in the main README?

Yes, please :)

File: README.md [hunk 1]
```diff
@@ -44,14 +44,15 @@
 
 The Raiden Light Client SDK is a [Raiden Network](https://raiden.network) compatible client written in JavaScript/Typescript, capable of running in modern web3-enabled browsers, wallets and Node.js environments.
 
+The Raiden CLI aims to provide a HTTP REST server that is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
```
Comment (christianbrb) [inline README.md:47]:
```suggestion
The Raiden CLI provides an HTTP REST server that is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
```

File: README.md [hunk 2]
```diff
@@ -7,12 +7,12 @@
       alt='' 
       src="https://user-images.githubusercontent.com/35398162/54018436-ee3f6300-4188-11e9-9b4e-0666c44cda53.png" /></a>
   <br/>
-  Raiden Light Client SDK and dApp
+  Raiden Light Client SDK, CLI and dApp
```
Comment (christianbrb) [inline README.md:10]:
```suggestion
  Raiden Light Client SDK, CLI/API and dApp
```

File: README.md [hunk 3]
```diff
@@ -7,12 +7,12 @@
...
   <br/>
 </h2>
 
 <h4 align="center">
-  JavaScript SDK and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
+  JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
```
Comment (christianbrb) [inline README.md:15]:
```suggestion
  JavaScript SDK, CLI/API and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
```

File: README.md [hunk 4]
```diff
@@ -44,14 +44,15 @@
 
 The Raiden Light Client SDK is a [Raiden Network](https://raiden.network) compatible client written in JavaScript/Typescript, capable of running in modern web3-enabled browsers, wallets and Node.js environments.
 
+The Raiden CLI aims to provide a HTTP REST server that is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
+
 The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Light Client SDK, which can be used with web3 wallets like [Metamask](https://metamask.io/) (Desktop) or [imToken](https://token.im/download) (mobile).
 
-> **INFO:** The Light Client SDK and dApp are **work in progress** and can only be used on the Ethereum **Testnets**.
+> **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects.
```
Comment (christianbrb) [inline README.md:51]:
```suggestion
> **INFO:** The Light Client SDK, CLI/API and dApp are all **work in progress** projects.
```

File: README.md [hunk 5]
```diff
@@ -44,14 +44,15 @@
 
 The Raiden Light Client SDK is a [Raiden Network](https://raiden.network) compatible client written in JavaScript/Typescript, capable of running in modern web3-enabled browsers, wallets and Node.js environments.
 
+The Raiden CLI aims to provide a HTTP REST server that is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
+
 The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Light Client SDK, which can be used with web3 wallets like [Metamask](https://metamask.io/) (Desktop) or [imToken](https://token.im/download) (mobile).
```
Comment (christianbrb) [inline README.md:49]: The CLI/API is also a reference application.

File: README.md [hunk 6]
```diff
@@ -64,6 +65,7 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
 - [Run Repository Code](#run-repository-code)
   - [Prerequisites](#prerequisites)
   - [SDK Documentation](#sdk-documentation)
+  - [CLI Documentation](#cli-documentation)
```
Comment (christianbrb) [inline README.md:68]:
```suggestion
  - [CLI/API Documentation](#cli-api-documentation)
```

File: README.md [hunk 7]
```diff
@@ -74,32 +76,32 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
...
 
-The Raiden client code is available [here](https://github.com/raiden-network/raiden) and has been [released for mainnet](https://medium.com/raiden-network/red-eyes-mainnet-release-announcement-d48235bbef3c) with a limited alpha release of the Raiden Network in December 2018.
+The Raiden client code is available [here](https://github.com/raiden-network/raiden) and has been [released for mainnet](https://medium.com/raiden-network/alderaan-mainnet-release-announcement-7f701e58c236).
 
 <center>
 <img 
       width='750px' 
       alt='' 
-      src="https://user-images.githubusercontent.com/43838780/67964837-438d2080-fc01-11e9-93a7-c38129b7d566.png" />
+      src="https://user-images.githubusercontent.com/43838780/85526750-8ecdc680-b60a-11ea-8a42-07da6b0f8296.png" />
```
Comment (christianbrb) [inline README.md:85]: Nice 👍

File: README.md [hunk 8]
```diff
@@ -74,32 +76,32 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
...
 
 The goal of the Raiden Light Client SDK is to provide an easy-to-use framework, which can be integrated by any JavaScript developer. The SDK will simplify the process of embedding and using the Raiden Network for token transfers in decentralized applications
 
-With the SDK we want to make your life as a dApp dev easier:
+With the SDK we want to make your life as a dApp developer easier by:
 
-- You should be able to interact with the Raiden Network easily with your dApp.
-- We want to help you to enable your users to make token transfers using their consumer wallets like imToken or Metamask.
-- It should be possible to send tokens using low end devices, which would not be capable of running a full Raiden node.
+- Providing an easy way for your dApp to interact with the Raiden Network.
```
Comment (christianbrb) [inline README.md:92]: I would like to keep it in an active voice

see: https://developers.google.com/style/voice

File: README.md [hunk 9]
```diff
@@ -74,32 +76,32 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
...
+- Making it possible to transfer and receive tokens using low end devices.
 
 ## Architecture
 
 ### [Raiden Light Client SDK](./raiden-ts/README.md)
 
 This is a standalone Typescript library which contains all the low level machinery to interact with the Ethereum blockchain and the Raiden Network.
 
-Its target audience is blockchain and dApp developers looking into interacting with and performing transfers through the Raiden Network from their apps. Targeting browsers and Node.js as initial platforms allows it to reach the majority of current and in-development dApps, as well as to work as a common language reference implementation for ports and re-implementations in other future languages and environments.
+Its target audience is blockchain and dApp developers looking to interact with the Raiden Network from their apps. Targeting browsers and Node.js as initial platforms allows the SDK to reach a majority of the current and in-development dApps, as well as to work as a common language reference implementation for ports and re-implementations in other languages and environments.
```
Comment (christianbrb) [inline README.md:102]: Nitpick: Performing transfer is more relevant than interacting ;)

File: README.md [hunk 10]
```diff
@@ -227,6 +230,10 @@ To run the code in this repository, you must have Node.js 10+ on your computer a
 
 Go to the [SDK Documentation](https://lightclient.raiden.network/docs/) for more information on how to install and use the SDK.
 
+### CLI Documentation
```
Comment (christianbrb) [inline README.md:233]:
```suggestion
### CLI/API Documentation
```
Check if the reference in the table of contents still works with my change. Thanks :)

File: README.md [hunk 11]
```diff
@@ -227,6 +230,10 @@ To run the code in this repository, you must have Node.js 10+ on your computer a
...
+
+Go to the [CLI README](https://github.com/raiden-network/light-client/tree/master/raiden-cli) for installation instructions and the current status of the API.
```
Comment (christianbrb) [inline README.md:235]:
```suggestion
Go to the [CLI/API README](https://github.com/raiden-network/light-client/tree/master/raiden-cli) for installation instructions and the current status of the API.
```

**Pull request merged**